tests/eas/acceptance.py

```diff
@@ -28,15 +28,15 @@
 from env import TestEnv
 from test import LisaTest, experiment_test
 
-# Read the config file and update the globals
```

Comment:
Let's keep this patch before the previous one... so that we have all the tests re-factoring one after the other...